javascript/javascript1/week3/hw3.js

```diff
@@ -0,0 +1,151 @@
+// Item array removal
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+names.splice(1,1);  // Remove the item using index value directly
+
+for(let i = 0; i < names.length; i++)  // UNKNOWN INDEX USE FOR LOOP
+{
+  if(names[i] == nameToRemove){
+    names.splice(i, 1);
+    break;
+  }
+}
+console.log(names);
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+//........................................................//
+// Traveltime
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+function totalTime(travelInformation){
+   let time = travelInformation.destinationDistance/travelInformation.speed;
+  let  hours = Math.floor(time);
```

Comment:
Variable naming conventions , hour/minute - is a general naming, be very specific.
ex: travelInHour etc...